build_tools/benchmarks/common/benchmark_presentation.py

```diff
@@ -75,6 +83,21 @@ def get_current_and_base_value(self, obj: T) -> Tuple[int, Optional[int]]:
     """Returns the current and base (can be None) value."""
     raise NotImplementedError()
 
+  def get_series_id(self, benchmark_id: str) -> str:
+    """Returns the dashboard series id."""
+    # TODO(#11076): Remove legacy path.
+    # Whitespace is used in the legacy benchmark id as the delimiter, while not
+    # used in the new benchmark id. This is a temporary solution to generate
+    # both ids during the migration.
+    if " " in benchmark_id:
+      return self.get_series_name(benchmark_id)
+    return f"{benchmark_id}-{self.get_metric_id()}"
```

Comment:
Wait the metric being collected seems like it should be part of the benchmark definition. What you're measuring is kind of the whole thing with benchmarks

Reply:
We have `CompilationStatistics` and `BenchmarkResults` in benchmark definition to represent the metric data. Here is about how to present the data to the dashboard. Since the dashboard doesn't support structural metrics data (e.g. JSON), we have to flatten everything.

So the metrics in compilation statistics are extracted here and become separate series. The part should be dropped once we have a better database to store the data.